corehq/apps/app_manager/suite_xml/post_process/remote_requests.py

```diff
@@ -147,7 +148,8 @@ def build_case_id_query_data(self):
         if self.module.is_multi_select():
             data.ref = "."
             data.nodeset = self._get_multi_select_nodeset()
-            data.exclude = self._get_multi_select_exclude()
+            if not case_search_sync_cases_on_form_entry_enabled_for_domain(self.domain):
+                data.exclude = self._get_multi_select_exclude()
```

Comment:
`data.exclude` doesn't look like it gets set if this feature _is_ enabled. Is that okay?

Reply:
yes this was a modification after a QA bug arose